rules/cross-platform/defense_evasion_agent_spoofing_multiple_hosts.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2021/07/14"
+maturity = "production"
+updated_date = "2021/07/14"
+min_stack_version = "7.14.0"
+
+[rule]
+author = ["Elastic"]
+description = """Detects when multiple hosts are using the same agent ID. This could occur in the event of an agent
+being taken over and used to inject illegitimate documents into an instance as an attempt to spoof events in order to
+masquerade actual activity to evade detection.
+"""
+false_positives = [
+    """
+    This is meant to run only on datasources using agents v7.14+ since versions prior to that will be missing the
+    necessary field, resulting in false positives.
+    """,
+]
+from = "now-9m"
+index = ["logs-*", "metrics-*", "traces-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Agent Spoofing - Multiple Hosts Using Same Agent"
+risk_score = 73
+rule_id = "493834ca-f861-414c-8602-150d5505b777"
+severity = "high"
+tags = ["Elastic", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "threshold"
+
+query = '''
+event.agent_id_status:*
```

Comment:
is it possible to make checks on the agent version?

Reply:
I originally had that in the query, but agent versions can get quite wonky (as I saw in my observations) and I could not come up with a reliable way to parse it out and cast it for comparison (`between` with EQL could get you close with `number` casting, but I don't think it is worth the complexity)

Reply:
fair enough. i think `regex` would work too and still be efficient. we can come back to it if we get complaints about FPs